data/Harvey_2017/metadata.yml

```diff
@@ -0,0 +1,172 @@
+source:
+  primary:
+    key: Harvey_2017
+    bibtype: Article
+    year: '2017'
+    author: J. M. Harvey and A. J. M. Hopkins and M. A. Langley and C. R. Gosper and
+      M. R. Williams and C. J. Yates
+    journal: Australian Journal of Botany
+    title: Long-term studies of post-fire reproduction in an Australian shrubland
+      and woodland
+    volume: '65'
+    number: '4'
+    pages: '339'
+    doi: 10.1071/bt17011
+contributors:
+  data_collectors:
+  - last_name: Gosper
+    given_name: Carl
+    ORCID: 0000-0002-0962-5117
+    affiliation: Biodiversity and Conservation Science, Department of Biodiversity,
+      Conservation and Attractions, WA, Australia
+    additional_role: contact
+  - last_name: Harvey
+    given_name: Judith
+    ORCID: .na
+    affiliation: Biodiversity and Conservation Science, Department of Biodiversity,
+      Conservation and Attractions, WA, Australia
+  austraits_curators: Elizabeth Wenk
+dataset:
+  data_is_long_format: no
+  custom_R_code: '
+    data %>%
+      mutate(Comeditive_stratum = ifelse(PFT_Gosper2012 == "Ephem","post_fire_ephemeral",Comeditive_stratum))
+  '
+  collection_date: 1985/2015
+  taxon_name: Species name
+  location_name: location
+  description: An investigation of patterns of reproduction in south-western Australian
+    kwongan and woodland after prescribed fires with a 30-year longitudinal study,
+    with the aim of identifying appropriate fire intervals.
+  basis_of_record: field
+  life_stage: adult
+  sampling_strategy: Experimental fires ^p Two areas of kwongan and mixed Allocasuarina
+    huegeliana and Eucalyptus wandoo woodland vegetation were burnt at the end of
+    the summer dry season, in April 1985 and March 1990. The 1985 fire in a 44-ha
+    area of kwongan and woodland (previously burnt in 1960) was of moderate intensity,
+    hot in patches, and resulted in near-complete burning of the target area. The
+    1990 fire, in an area of 108 ha (previously burnt in 1965), was of moderate to
+    high intensity. It burnt through most of the kwongan at a rate of 1100mh-1, through
+    Allocasuarina woodlands in a continuous front with flame height reaching 2m and
+    at a rate of 200mh-1. In Eucalyptus woodland, the fire slowed down to 100mh-1
+    because of lower levels of ground cover and litter and reduced exposure to wind.
+    The canopy of the Eucalyptus woodland and some of the Allocasuarina woodland remained
+    unburnt; there were occasional unburnt mature Allocasuarina plants in areas where
+    the fire had been of low intensity, but the shrub and herb layers were mostly
+    or completely burnt. ^p Sampling strategy ^p To understand post-fire vegetation
+    recovery and dynamics, 21 100-m2 circular plots (14 in woodland and seven in kwongan)
+    were established in the spring of 1983, before the 1985 autumn fire, and resampled
+    in the spring of 1985, 1986, 1987, 1988, 1990, 1992, 1995, 2000 and 2015. Similarly,
```

Comment:
A bit of overlap in the sampling_strategy here and the methods for each trait

Reply:
Yes, that is intentional, since it describes both the strategy and the method, so I didn't want to omit it from either field.